getHacks.js

```diff
@@ -6,26 +6,33 @@ const url = `mongodb://${process.env.MONGO_USER}:${process.env.MONGO_PASSWORD}@d
 
 /**
  * Lists all Hacks currently stored in the database
- * @param {*} event 
- * @param {*} context 
+ * @param {*} event
+ * @param {*} context
  */
 export const list = async (event, context) => {
-    context.callbackWaitsForEmptyEventLoop = false;
-    if(conn == null){
-        conn = await mongoose.createConnection(url,{bufferCommands:false,bufferMaxEntries:0});
-    }
+  context.callbackWaitsForEmptyEventLoop = false;
+  if (conn == null) {
+    conn = await mongoose.createConnection(url, {
```

Comment:
How many times will this function get called? will the connection ever get reused?

Reply:
We will find a way to see if we can share a mongoose connection or find out a way so the connection is only called once across all the lambda functions.

Reply:
Sorry, that wasn't my intention. My question was about whether we need to guard against it being null instead of always creating a new connection.

Reply:
The guard is needed because Lambda may or may not use the same container every time the function is called. so if the connection is null it can initiate it again, however if it is not then the function does not need to spend compute time re-establishing the connection.